php/libraries/NDB_BVL_Feedback.class.inc

```diff
@@ -1,4 +1,4 @@
-<?php
+<?php declare(strict_types=1);
```

Comment:
❤️

Reply:
It might break things though...

Reply:
My Faith is in Travis.

Reply:
Your faith might break things, though.

Reply:
Well, after a quick test, instrument_list is broken
```
PHP Fatal error:  Uncaught TypeError: Argument 3 passed to NDB_BVL_Feedback::setup() must be of the type string or null, int given, called in /var/www/loris_1/php/libraries/NDB_BVL_Feedback.class.inc on line 102
```
I will remove the strict_types

Reply:
😿